plugin/pom.xml

```diff
@@ -15,4 +15,21 @@
         <jenkins.version>2.60.3</jenkins.version>
         <java.level>8</java.level>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
In Jenkins plugins we commonly use java.util.logging.
If you want to use SLF4j, note that the libraries already come from the core. See the plugin POM: https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L260-L286

Reply:
Kafka logger uses SLF4j.

Reply:
It can be removed from here anyway, because `kafka-client`already declares it.